### ISSUE TITLE
Add an OTel training+certs showcase

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#3463]
OTel training+certs will be available in the next couple of months. We'll need to add a section or page to showcase them. 

@caniszczyk shared: "Something like https://prometheus.io/support-training/".

References:

- https://kubernetes.io/training/
- https://www.cncf.io/training/certification/otca/
- https://www.cncf.io/training/courses/?_sft_lf-project=opentelemetry
- https://github.com/cncf/techdocs/issues/197

Old:

~Maybe also rework vendors? (Cf. #2178)~

**Comment by zzadxz** [comment]
Hey! Can I take this one?

**Comment by svrnm** [comment]
Hey @zzadxz, 

since we are still waiting for the trainings+certs being available within the next couple of months, this is not something you can help with right now. Take a look at the issues we have marked with "help wanted" or "good first issue" to find some where you can start immediately

**Comment by tiffany76** [comment]
Any thoughts on where this page should be located and what it should be called? Based on the Prometheus and CNCF sites, I was thinking of adding **Training** to the top nav. **Support and Training** seems a little too long to fit comfortably.

**Comment by chalin** [comment]
I like your proposal.

If we find that our top nav is too busy, we could fold Ecosystem into / to be under Community. Thoughts?

**Comment by tiffany76** [comment]
Let me finish creating the page and I'll post a screenshot of the preview here. I agree Ecosystem could fit under Community if things look too busy.

**Comment by tiffany76** [comment]
Ignore the poor image formatting. My screen resolution is 1920x1200. I think the nav looks okay?

![Image](https://github.com/user-attachments/assets/22ea062b-91ac-4f03-a7c3-35b52918d3b2)

**Comment by chalin** [comment]
Yes, it looks good. We'll have to check it at various widths to ensure that it always fits. Thanks!
